Source/WebKit/Platform/spi/Cocoa/AuthenticationServicesCoreSPI.h

```diff
@@ -275,6 +275,10 @@ typedef NS_ENUM(NSInteger, ASCredentialRequestStyle) {
 @property (nonatomic) ASCredentialRequestStyle requestStyle;
 
 @property (nonatomic, nullable, copy) ASGlobalFrameIdentifier *globalFrameID;
+
+#if TARGET_OS_IOS
```

Comment:
We should add a named macro in `PlatformHave.h` for this property.

Note that the equivalent of `TARGET_OS_IOS` is `PLATFORM(IOS) || PLATFORM(MACCATALYST)`.

Reply:
This is a forward declaration of SPI and the macro matches the source declaration. Defining it differently is likely to cause us trouble down the road.

Reply:
We generally only use `TARGET_OS_` macros in framework headers. Using `HAVE` is the approach we use in other `...SPI.h` headers.

`TARGET_OS_IOS` == `PLATFORM(IOS) || PLATFORM(MACCATALYST)` in WebKit. Many more things would break if that were to change.

@hortont424 may have thoughts.

Reply:
`HAVE()` macros would definitely be the right approach if this declaration was from a set of related declarations for a whole feature that may be unavailable on different platforms. But that's not the case here :)

This whole file is a forward declaration of a series of framework headers, and right now is textually identical to the source files it's redeclaring, which makes updates as easy as copy/paste. If we switch to using piecemeal `HAVE()` sprinkled throughout, in the best case we'd just be defining those macros to use `TARGET_OS_...` anyway (because that's how the source file declares them) and introducing a new maintenance burden to updating to this file. However introducing discrepancies in the macros used between this file and the source files makes it much easier for them to deviate and introduce bugs in the future.

Reply:
Sounds good! It turns out we don't have strict rules about this case, so this is fine.

Though I would like to note that it may not always be as easy as copy/paste. In this case, we're fine because we don't need to support building on older iOS versions, where the SPI may not exist. However, if we're adopting newer SPI on a particular macOS version, we would need a `HAVE`.